enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -81,3 +89,20 @@ def _get_course_overview(self, course_run_id):
                 return overview
 
         return None
+
+    def _get_course_run_url(self, request, course_run_id):
+        """
+        Get the appropriate course url while incorporating Executive Education content.
+        """
+        course_key = CourseKey.from_string(course_run_id)
+        course_key_str = '{}+{}'.format(course_key.org, course_key.course)
+        try:
+            course_data = get_course_catalog_api_service_client().get_course_details(course_key_str)
```

Comment:
This is going to call the Discovery service once for every enterprise course enrollment in each request to fetch all enterprise enrollments for a learner - right?
We should at least cache this (the correct course URL, perhaps dates, too?) in memcached, since it's data that presumably would rarely change.
Medium term, maybe we could build some `EnterpriseCourseDetails` model or something that keeps minimal information about content - like the key, course run url, dates, and whatever other minimal stuff is required to render correct data on the B2B learner portal dashboard.